docs/sidebar.json

```diff
@@ -4,6 +4,7 @@
   "early-access",
   {
     "Start Building": [
+      "start-building/create-project",
       "start-building",
```

Comment:
Move below "start building"

Reply:
I put it before the "Choose Your Technology" document, because you need to `Create A Project` before you start on the implementation.

> What's next?
> You created your first Ory Cloud Project. Now it's time to start building!

Do you think it should come after?

Reply:
True, most people would however not really struggle with creating a project and probably want to start building immediately. You could definitely link it in "start-building" though!

Might also make sense to move it to "guides"

Reply:
Fair enough! 
I will leave it in `start-building` for now. 
I see `start-building` as the same category of documentation as `guides` only more "basic" guides. Always trying to follow [this format](https://github.com/ory/docs/#categories)!